### PR TITLE
Added PROXY_ADDRESS_FORWARDING for Keycloak

### PR DESCRIPTION
**Pull request by seriva**
Added PROXY_ADDRESS_FORWARDING for Keycloak

### CODE REVIEW

File: ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2 [hunk 1]
```diff
@@ -209,6 +209,10 @@ spec:
               name: "{{ auth_service_name }}-db"
         - name: X509_CA_BUNDLE
           value: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+{% if data.service.proxy_address_forwarding is defined and data.service.proxy_address_forwarding %}
```
Comment (atsikham) [inline ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2:212]: `if data.service.proxy_address_forwarding is defined` - should it be checked if there is alvays a default value?

**Review by atsikham — CHANGES_REQUESTED** [review]

File: ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2 (hunk 1 above)
Comment (seriva) [reply ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2:212]: From Epicli perspective this default should always be there but when we move to the collections as people might chose not to run through CLI.  So I like to cover the bases.

Comment (atsikham) [reply ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2:212]: default(false) filter could be used, but current version is of for me

**Review by atsikham — DISMISSED** [review]

**Review by to-bar — DISMISSED** [review]

**Comment by przemyslavic** [comment]
@seriva I think it can be merged after the conflict is resolved as it has already been tested by @plirglo https://github.com/epiphany-platform/epiphany/issues/2785#issuecomment-1000101017.

**Review by erzetpe — APPROVED** [review]

**Review by atsikham — APPROVED** [review]

**Pull request merged**